server/job/job.js

```diff
@@ -20,11 +20,11 @@ import { JobEvent } from './jobEvent'
  * -- canceled
  *
  * Methods that can be overwritten by subclasses:
- * - onStart
- * - execute
- * - beforeSuccess
- * - beforeEnd
- * - onEnd
+ * - onStart (in tx)
+ * - execute (in tx)
+ * - beforeSuccess (in tx)
+ * - beforeEnd (in tx)
+ * - onEnd (out of tx)
```

Comment:
nice

Reply:
yes, just to explain a bit what are the methods executed under transaction in the job